src/components/dashjs-api-control-element/dashjs-api-control-element.tsx

```diff
@@ -65,41 +48,66 @@ export class DashjsAPIControlElement {
       });
       toast.present();
     } else {
-      this.valueChanged.emit(this.functionValue);
+      this.valueChanged.emit(functionValue);
     }
   }
 
   render() {
-    const control = [];
+    let functionValue = [];
+    let control = [];
+
+    for (let i = 0; i < this.param.length; i++) {
```

Comment:
fox fox sake! Use `this.param.length.forEach(elm => { /*yourcode*/ });`

Reply:
but I like for lopps :)